src/web/client/common/authenticationProvider.ts

```diff
@@ -31,6 +31,7 @@ export async function dataverseAuthentication(dataverseOrgURL: string): Promise<
                 sendErrorTelemetry(telemetryEventNames.WEB_EXTENSION_NO_ACCESS_TOKEN);
             }
         }
+        sendInfoTelemetry("Auth", { 'web.userId': session.id });
```

Comment:
check about data tagging/any other compliance before merging the PR.

Reply:
@tehcrashxor We are planning to log userId as part of this telemetry change. Any GDPR concerns here? We already log this id in other code paths, and this is compliant, correct?

Reply:
@tyaginidhi I think we should replace session.id(identifier of the session) with session.account.id(unique identifier of the account). Can you once test locally to check what is the value for this...must be a GUID but is it encoded/hashed.
Maybe you can verify it using: https://unify.services.dynamics.com/CRM/Org/<orgId>/Organization/ApplicationUsers

For compliance we will have to check with @tehcrashxor

Reply:
We should not use any of the session field in logging. Verified on my local that all of this is PII:
![image](https://user-images.githubusercontent.com/6157891/198230400-acc117cf-9e68-4595-a2f0-d3afc1a2e907.png)